From 5a7cbb0d509c8b833597aa541797a3fabac362cd Mon Sep 17 00:00:00 2001
From: Yuchou Gan <yuchou.gan@nxp.com>
Date: Thu, 26 Apr 2018 18:52:58 +0800
Subject: [PATCH 3624/5242] MGS-3846 [#imx-1015] fix coverity high impact
 issue

commit  707c11791659df16e36e9640ee38fc7251743491 from
https://source.codeaurora.org/external/imx/linux-imx.git

Using uninitialized value minf when calling snprintf.
Initilize it to fix.

Date: 25th Apr, 2018
Signed-off-by: Yuchou Gan<yuchou.gan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../freescale/gc_hal_kernel_platform_imx.c         |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/mxc/gpu-viv/hal/os/linux/kernel/platform/freescale/gc_hal_kernel_platform_imx.c b/drivers/mxc/gpu-viv/hal/os/linux/kernel/platform/freescale/gc_hal_kernel_platform_imx.c
index 39197db..0bc059c 100644
--- a/drivers/mxc/gpu-viv/hal/os/linux/kernel/platform/freescale/gc_hal_kernel_platform_imx.c
+++ b/drivers/mxc/gpu-viv/hal/os/linux/kernel/platform/freescale/gc_hal_kernel_platform_imx.c
@@ -324,6 +324,7 @@ static ssize_t gpu3DMinClock_show(struct device_driver *dev, char *buf)
 
     galDevice = platform_get_drvdata(pdevice);
 
+    minf = 0;
     if (galDevice->kernels[gcvCORE_MAJOR])
     {
          gckHARDWARE_GetFscaleValue(galDevice->kernels[gcvCORE_MAJOR]->hardware,
-- 
1.7.9.5

